airbyte-server/src/main/java/io/airbyte/server/converters/JobConverter.java

```diff
@@ -99,7 +100,8 @@ public static AttemptRead getAttemptRead(Attempt attempt) {
 
   public static LogRead getLogRead(Path logPath) {
     try {
-      return new LogRead().logLines(IOs.getTail(LOG_TAIL_SIZE, logPath));
+      var logs = LogHelpers.getJobLogFile(new EnvConfigs(), logPath);
```

Comment:
I just realised it's cleaner to save the cloud path as part of the log path when we save the job attempt - will make this change tomorrow.

Reply:
I'm punting on this. The logic for this winds through several classes in temporal and the scheduler. I think this will be an easier task once we are on a proper database. Will create a ticket to do so.